### PR TITLE
1569 temp branch

### PR DESCRIPTION
**Pull request by splekhanov**
**DO NOT MERGE!**
Travis testing temporary branch. Can be deleted after the experiments

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/BrowserOpen.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4683015&bid=15662044)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java [hunk 1]
```diff
@@ -86,6 +83,7 @@ private static String driverPath(String driverName) {
     public static String  LATEST_VERSION = "LATEST";
     public static String DRIVER_VERSION = LATEST_VERSION;
     public static String PRELATEST_VERSION = "PRELATEST";
+    public static String ARGUMENTS_PROPERTY = "arguments";
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4683015)

**Comment by mf-code** [comment]
[4095] - https://transfer.sh/ZC05M/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4095] - Allure report on Netlify: https://5df78ddf5eec139e060f63b6--musing-swartz-6fc060.netlify.com

**Pull request closed without merging**